### DISCUSSION TITLE
UpdateHub v2.4.1

### DISCUSSION THREAD

**Discussion by onyx-nxt** [NexovaDev/UpdateHub#2111] [Announcements]
## Version
`2.4.1`

# Changelog
- Fix broken OS updates that would cause UpdateHub to not start
- Fix some special characters in outdated app names that would cause it to go undetected
- Fix broken icons/assets
- Better update flow error handling
- And more bug fixes!

# How to update
Click the Update Available button in the sidebar. (if you don't see it, force close the app by pressing the shift key, clicking the X icon in the windows title bar, and then relaunching the app.)

<img src="https://github.com/NexovaDev/UpdateHub/assets/71038229/d190f605-897d-4e6f-b147-fcccb07bdfe2" width="584" /> \
Then wait for the update file to download, and it will install automatically. You now have the latest version!

### VirusTotal scan
This app is not a virus, trust me. If you want, here's a [VirusTotal scan](https://www.virustotal.com/gui/file/811dfd3768fc791bc0dfd3b6c1fd9ee01abb3fa5687b4b3be34c1179aa458eda/ "VirusTotal Scan"). As you can see, no anti-virus software has flagged it as a virus. The executable below's SHA-256 Hash is `811DFD3768FC791BC0DFD3B6C1FD9EE01ABB3FA5687B4B3BE34C1179AA458EDA`, which matches the VirusTotal scan.

### Remember, if you encounter any bugs in UpdateHub, report them so we can fix them!

The one-click installer is attached below.

### SHA256
`811DFD3768FC791BC0DFD3B6C1FD9EE01ABB3FA5687B4B3BE34C1179AA458EDA`

<hr /><em>This discussion was created from the release <a href='https://github.com/NexovaDev/UpdateHub/releases/tag/v2.4.1'>UpdateHub v2.4.1</a>.</em>

**Comment by temis01** [comment]
Hi great app, it could successfully update stuff that wingetui couldn't !

Setup needs an "OK" to install button before it does anything to the OS ... and warn the user that additional programs or updates will be installed.

in 2.4.1 everything is assuming and silent !

**Comment by Robert51** [comment]
UpdateHub v2.4.1 times out whenever it cannot find any program that needs updating. However, when UpdateHub v2.4.1 finds any one program that needs updating, it does not time out, and offers to update the said program.
